examples/web-react/src/demo.jsx

```diff
@@ -4,22 +4,24 @@ import { Button } from '@lmc-eu/spirit-web-react/components/Button';
 
 const Demo = () => (
   <section>
-    <h2>Button</h2>
-    <Button color="primary" onClick={() => console.log('Primary button clicked!')}>
-      Primary button
-    </Button>
-    <Button color="secondary" onClick={() => console.log('Secondary button clicked!')}>
-      Secondary Button
-    </Button>
-    <Button color="tertiary" onClick={() => console.log('Tertiary button clicked!')}>
-      Tertiary button
-    </Button>
-    <Button color="inverted" onClick={() => console.log('Inverted button clicked!')}>
-      Inverted button
-    </Button>
-    <Button color="danger" onClick={() => console.log('Danger button clicked!')}>
-      Danger button
-    </Button>
+    <div className="Container">
```

Comment:
You can use `<Container />` component.

Reply:
Not until it's released… 😐

Reply:
`<Container />` component has been released. Just check [changelog](https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/CHANGELOG.md).